src/reference/01-General-Info/02-Community-Plugins.md

```diff
@@ -255,6 +255,7 @@ your plugin to the list.
 - [safety-plugin](https://github.com/leobenkel/safety_plugin): Enforce the use of specified versions of dependencies across your company
 - [sbt-dependency-lock](https://stringbean.github.io/sbt-dependency-lock):
 generate dependency lockfiles and check for changes at build time.
+- [sbt-unzip](https://github.com/djice/sbt-unzip-plugin): Extract zip dependencies where you want in your project.
```

Comment:
Is this plugin publicly available on Maven Central?

Reply:
Hi, yes this is pushed on maven central repository -> https://search.maven.org/artifact/io.github.djice/sbt-unzip/1.0.0/jar